src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      data-qa="big"
+      method="GET"
+      >
+      <label
+        for="largeInput"
+        class="large__bar"
+        >
```

Comment:
```suggestion
      <label for="largeInput" class="large__bar">
```
You don't really need `for` attribute, as you wrap input into the label